extensions/barrage/src/main/java/io/deephaven/extensions/barrage/chunk/BooleanChunkInputStreamGenerator.java

```diff
@@ -132,9 +130,9 @@ public int drainTo(final OutputStream outputStream) throws IOException {
 
             // write the included values
             subset.forAllRowKeys(row -> {
-                final byte byteValue = chunk.get((int) row);
-                if (byteValue != NULL_BYTE) {
-                    context.accumulator |= (byteValue > 0 ? 1L : 0L) << context.count;
+                final Boolean value = BooleanUtils.byteAsBoolean(chunk.get((int) row));
```

Comment:
I don't know if it makes sense to include an isTrue method, then we don't need to actually create the boolean and the number of conditions should remain the same.

Reply:
It shouldn't actually create a Boolean, but reuse the singletons Boolean.TRUE and FALSE?